Dockerfile

```diff
@@ -1,17 +1,11 @@
-FROM golang:1.17-alpine
-
-LABEL email="atish.iaf@gmail.com"
-
+FROM golang:1.17 AS builder
 EXPOSE 8080
-
 RUN mkdir /app
-
 ADD . /app
-
 WORKDIR /app
+LABEL email="atish.iaf@gmail.com"
+RUN CGO_ENABLED=0 GOOS=linux go build -o main ./
 
-RUN go mod download
-
-RUN go build -o main 
-
-CMD [ "/app/main" ]
\ No newline at end of file
+FROM alpine:latest AS production
+COPY --from=builder /app .
```

Comment:
**Suggestion:** In your Dockerfile, you are copying all files from the builder stage to the production stage. This could include unnecessary files and increase the size of the final image. You should only copy the necessary files.
```suggestion
COPY --from=builder /app/main .
```